source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd

```diff
@@ -0,0 +1,30 @@
+# Definities en afkortingen
+
+**Avimap:** Avimap is een digitale tool ontwikkeld door SOVON voor het uitvoeren van territoriumkartering van broedvogels in het veld.
+
+**code repository**: Een code repository is een digitale opslagplaats waar de broncode van softwareprojecten wordt beheerd en opgeslagen, vaak met versiebeheer om wijzigingen bij te houden.
+Het stelt ontwikkelaars in staat om samen te werken, wijzigingen te volgen en terug te keren naar eerdere versies van de code.
+
+**CRS:** Een CRS, of Coordinate Reference System, is een systeem dat wordt gebruikt om geografische locaties op de aarde te definiëren.
+Een CRS beschrijft hoe tweedimensionale of driedimensionale ruimtelijke gegevens worden gepositioneerd, en hoe ze worden geprojecteerd op een plat vlak.
+Amersfoort/RD New (EPSG:28992) is een voorbeeld van een lokale CRS, specifiek voor Nederland, dat gebruikmaakt van een projectie die is geoptimaliseerd voor het Nederlandse grondgebied.
+
+**MAS:** Het MAS, of Meetnet Agrarische Soorten, monitort broedvogels (en zoogdieren) in agrarische gebieden om trends in aantallen en verspreiding vast te stellen, en de effectiviteit van beschermingsmaatregelen te evalueren [@MAS].
+Dit meetnet wordt reeds toegepast in Nederland en sinds de opstart van het MBAG project ook in Vlaanderen.
+
+**MBAG:** Het MBAG project, of Meetnet Biodiversiteit in het Agrarisch Gebied, is een initiatief van het Instituut voor Natuur- en Bosonderzoek (INBO) in Vlaanderen dat gericht is op het monitoren en analyseren van de biodiversiteit in landbouwgebieden.
+Het project richt zich op het verkrijgen van inzicht in de biodiversiteit, met een focus op akkervogels, bestuivers, bodembiodiversiteit, en landschapselementen, om zo bij te dragen aan zowel natuurbehoud als duurzame landbouwpraktijken [@MBAG].
+
+**QGIS:** QGIS is een open-source geografisch informatiesysteem waarin je geografische gegevens importeren, bekijken, bewerken, analyseren en visualiseren [@QGISSoftware].
```

Comment:
...waarin je geografische gegevens **kan** importeren...

Reply:
What do you mean with "the first two links"?

Reply:
@EmmaCartuyvels1 kan je hier nog op terugkomen of approven?